Octokit/Models/Request/CreateFileRequest.cs

```diff
@@ -106,7 +106,7 @@ public class CreateFileRequest : ContentRequest
         /// Creates an instance of a <see cref="CreateFileRequest" />.
         /// </summary>
         /// <param name="message">The message.</param>
-        /// <param name="content">The content.</param>
+        /// <param name="content">The content encoded in base64.</param>
```

Comment:
@ryangribble @Haacked how do we feel about this breaking change? Should we instead introduce overloads?